helm/cloud-provider-vsphere/charts/kube-vip/values.yaml

```diff
@@ -0,0 +1,96 @@
+# Default values for kube-vip.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: ghcr.io/kube-vip/kube-vip
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.5.11"
+
+config:
+  address: ""
+
+env:
+  vip_interface: "eth0"
+  vip_arp: "true"
+  lb_enable: "true"
+  lb_port: "6443"
+  vip_cidr: "32"
+  cp_enable: "false"
+  svc_enable: "true"
+  vip_leaderelection: "false"
+
+envFrom: []
+  # Specify an externally created Secret(s) or ConfigMap(s) to inject environment variables
+  # For example an externally provisioned secret could contain the password for your upstream BGP router, such as
+  #
+  # apiVersion: v1
+  # data:
+  #   bgp_peers: "<address:AS:password:multihop>"
+  # kind: Secret
+  #   name: kube-vip
+  #   namespace: kube-system
+  # type: Opaque
+  #
+#- secretKeyRef:
+#    name: kube-vip
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+securityContext:
+  capabilities:
+    add:
+      - NET_ADMIN
+      - NET_RAW
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+# limits:
+#   cpu: 100m
+#   memory: 128Mi
+# requests:
+#   cpu: 100m
+#   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations:
```

Comment:
Possibly need master as well to support older versions

Reply:
Nice Catch!

Reply:
Updated.